### DISCUSSION TITLE
New-value on multiple Qselects

### DISCUSSION THREAD

**Discussion by verbowersock** [quasarframework/quasar#13409] [General - Components / Directives / etc] [answered]
I am new to Vue and Quasar so please be gentle. I am trying to figure out how to handle @new-value in Qselect with multiple qselects. Most of my data is structured as an array of objects with the following pattern {id: id, name: name}. Bypassing the event with  new-value-mode doesn't work. Passing the options array as an additional argument to the newValue function doesn't work either. The only way the new value appears in the dropdown is when I modify the options array directly in the function. Do I need to create separate functions for every one of my qselects that need the new value option?

**Comment by MilosPaunovic** [comment]
Is this something in the lines of what you need?
[https://codepen.io/milospaunovic/pen/ExQgdrm?editors=1011](https://codepen.io/milospaunovic/pen/ExQgdrm?editors=1011)

**Comment by verbowersock** [comment, marked as answer]
I actually decided to make separate functions for every q-select since they all will be calling different queries.
